src/ui.tsx

```diff
@@ -108,11 +108,7 @@ export class UserInterface extends Component<Props, State> {
     };
 
     this.annotationsObject.addAnnotation(this.state.activeTool);
-    this.presetLabels = this.props.presetLabels || [
-      "label-1",
-      "label-2",
-      "label-3",
-    ];
+    this.presetLabels = this.props.presetLabels || ["default-label"];
```

Comment:
```suggestion
    this.presetLabels = this.props.presetLabels || [];
```
I find having a label called "default-label" a bit confusing. How about leave `presetLabels` empty If a prop for `presetLabels` is  not available?